qa/suites/fs/mirror-ha/workloads/cephfs-mirror-ha-workunit.yaml

```diff
@@ -9,9 +9,15 @@ overrides:
 
 tasks:
   - exec:
-      client.1:
+      mon.a:
         - "ceph fs volume create dc"
         - "ceph fs volume create dc-backup"
+  # Remove volumes during unwind to avoid MDS replacement warnings:
+  - full_sequential_finally:
```

Comment:
I've not come across this before. This may be the first instance of 'full_sequential_finally' being used in ceph/qa ?

Reply:
I think so. It's been in teuthology.git for a long time.